consensus/init-bc1.sh

```diff
@@ -6,12 +6,13 @@ apk add curl
 
 echo "** ethiab ** Initializing Lodestar Beacon"
 
-node ./packages/cli/bin/lodestar beacon \
+exec node --max-old-space-size=4096 ./packages/cli/bin/lodestar beacon \
   --execution.urls "http://172.16.8.2:8551" \
   --jwt-secret "/root/data/execution/1/geth/jwtsecret" \
   --dataDir "/root/data/consensus/1" \
   --paramsFile "/root/consensus/config.yaml" \
   --genesisStateFile "/root/consensus/genesis.ssz" \
   --enr.ip 172.16.8.5 \
+  --nat true \
```

Comment:
So, it turns out that setting `--nat` in this case adds a new `warn` level message in the logs, because though it might not look like it, 172.16.8.5 is a local address.  It on the containers local interface.  I will use `--nat` on bc2 when I set the enr ip to the external IP address.  [source](https://github.com/ChainSafe/lodestar/blob/2cb067a7092f7589a8e455f35342f521a7ae48ec/packages/cli/src/cmds/beacon/initPeerIdAndEnr.ts#L69)

Re: the linked issue, I'm not sure what a "shared" mode docker network is, I wonder if they meant a `host` network?  In this case, we're using a non-default `bridge` network, so it's as if all of the current containers are on their own LAN subnet.

If you wouldn't mind removing that line I'll merge right away with my thanks.  I did notice the containers stopped faster.

Reply:
You are right, 172.16.0.0 to 172.31.255.255 is local ip range which means --nat does nothing hence the warning.

> I'm not sure what a "shared" mode docker network is, I wonder if they meant a host network?

Not 100% sure either but could be also an external network that already exists, this is similar to host network but can be any network the host has access to.

In compose has to be set to external
```yaml
networks:
  default:
    name: some_network
    external: true
```